html/index.html

```diff
@@ -550,6 +550,66 @@ <h1>Sampling and Sequencing Simulator</h1>
   throw new Error("failed to draw from cdf");
 }
 
+function daily_incidence_to_weekly_incidence(
+    daily_incidence, growth_factor) {
+  let weekly_incidence = 0;
+  let effective_incidence = daily_incidence;
+  for (let i = 0; i < 7; i++) {
+    weekly_incidence += effective_incidence;
+    effective_incidence /= growth_factor;
+  }
+  return weekly_incidence;
+}
+
+function individual_probability_sick(
```

Comment:
On the surface this looks reasonable, but it looks like it could in principle give a number greater than one, which suggests that it's wrong (possibly in a numerically unimportant way in practice). I'll think about it.

Reply:
On second thought, I think this is only because it assumes an exponential growth model, which breaks down when the total incidence gets high. Since that assumption is baked in everywhere and we warn people about it, I think that's fine.

Reply:
Documented in c7ca321